tests/test_cli.py

```diff
@@ -167,7 +167,7 @@ def test_browser_argument(browser_arg, capsys, platform):
                 )
 
 
-@pytest.mark.parametrize("platform", all_platform_fixtures, indirect=True)
+@pytest.mark.parametrize("platform", ['become_windows'], indirect=True)
```

Comment:
Why was this restricted to windows only?

Reply:
sorry, my bad, was changing this during fixing the csv sniff issue